### ISSUE TITLE
404 Not Found error at the end of the wizard

### ISSUE THREAD

**Issue by AtsushiSakai** [JuliaPackaging/BinaryBuilder.jl#1019]
Hi. Thank you for great OSS!!. 

I faced an issue when I create a GitHub token at the end of the wizard to create a PR for Yggdrasil.
After typing GitHub password, an Exception was thrown with 404 Not Found error like:
```
GitHub password:
>

┌ Warning: a SecretBuffer was `shred!`ed by the GC; use `shred!` manually after use to minimize exposure.
└ @ Base secretbuffer.jl:169
Error found in GitHub reponse:
	Status Code: 404
	Message: Not Found
	Docs URL: https://docs.github.com/rest
	Errors:
Stacktrace:
 [1] error(::String, ::String, ::String, ::String, ::String) at ./error.jl:42
 [2] handle_response_error(::HTTP.Messages.Response) at /Users/atsushisakai/.julia/packages/GitHub/Em3bH/src/utils/requests.jl:177
 [3] obtain_token(; ins::Base.TTY, outs::Base.TTY, github_api::GitHub.GitHubWebAPI) at /Users/atsushisakai/.julia/packages/BinaryBuilder/OMMHj/src/wizard/github.jl:87
 [4] obtain_token() at /Users/atsushisakai/.julia/packages/BinaryBuilder/OMMHj/src/wizard/github.jl:31
 [5] github_auth(; allow_anonymous::Bool) at /Users/atsushisakai/.julia/packages/BinaryBuilder/OMMHj/src/wizard/github.jl:18
 [6] yggdrasil_deploy(::String, ::VersionNumber, ::Array{BinaryBuilderBase.PatchSource,1}, ::String; open_pr::Bool, branch_name::Nothing) at /Users/atsushisakai/.julia/packages/BinaryBuilder/OMMHj/src/wizard/deploy.jl:138
 [7] yggdrasil_deploy(::BinaryBuilder.Wizard.WizardState, ::Bool) at /Users/atsushisakai/.julia/packages/BinaryBuilder/OMMHj/src/wizard/deploy.jl:127
 [8] _deploy(::BinaryBuilder.Wizard.WizardState) at /Users/atsushisakai/.julia/packages/BinaryBuilder/OMMHj/src/wizard/deploy.jl:235
 [9] step7(::BinaryBuilder.Wizard.WizardState) at /Users/atsushisakai/.julia/packages/BinaryBuilder/OMMHj/src/wizard/deploy.jl:267
 [10] run_wizard(::Nothing) at /Users/atsushisakai/.julia/packages/BinaryBuilder/OMMHj/src/Wizard.jl:116
 [11] run_wizard() at /Users/atsushisakai/.julia/packages/BinaryBuilder/OMMHj/src/Wizard.jl:86
 [12] top-level scope at REPL[2]:1
 [13] eval(::Module, ::Any) at ./boot.jl:331
 [14] eval_user_input(::Any, ::REPL.REPLBackend) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:134
 [15] repl_backend_loop(::REPL.REPLBackend) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:195
 [16] start_repl_backend(::REPL.REPLBackend, ::Any) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:180
 [17] run_repl(::REPL.AbstractREPL, ::Any; backend_on_current_task::Bool) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:292
 [18] run_repl(::REPL.AbstractREPL, ::Any) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:288
 [19] (::Base.var"#807#809"{Bool,Bool,Bool,Bool})(::Module) at ./client.jl:399
 [20] #invokelatest#1 at ./essentials.jl:710 [inlined]
 [21] invokelatest at ./essentials.jl:709 [inlined]
 [22] run_main_repl(::Bool, ::Bool, ::Bool, ::Bool, ::Bool) at ./client.jl:383
 [23] exec_options(::Base.JLOptions) at ./client.jl:313
 [24] _start() at ./client.jl:506
```

My Github password looks valid, so I don't know how to cope with this issue...
When I use "--deploy" option for build_tarball.jl, same thing was happened.

**Comment by giordano** [comment]
Yes, sorry, automatic creation of the pull request doesn't work anymore in the stable version of BinaryBuilder because of changes in GitHub authentication (#771).  This is fixed in the development version (#937) with a different authentication method, but this versions requires Julia v1.6-rc1 (we'll release a new version of BinaryBuilder when Julia v1.6 is officially released).

For the time being you should deploy to file and do a manual upload.

**Comment by AtsushiSakai** [comment]
oh. OK. It is known issue, and it will be fixed in Julia 1.6. Thank you for letting me know that!!
